File: client/src/main/scala/com/alphasystem/game/uno/client/ui/control/PlayingAreaView.scala
package com.alphasystem.game.uno.client.ui.control

import com.alphasystem.game.uno.client.ui.control.delegate.{PlayingAreaView => JPlayingAreaView}
import scalafx.scene.Node

class PlayingAreaView private(override val delegate: JPlayingAreaView) extends Node(delegate) {

}

object PlayingAreaView {
  def apply(): PlayingAreaView = new PlayingAreaView(new JPlayingAreaView())
}

File: client/src/main/java/com/alphasystem/game/uno/client/ui/control/delegate/PlayingAreaView.java
package com.alphasystem.game.uno.client.ui.control.delegate;

import javafx.scene.control.Control;
import javafx.scene.control.Skin;

public class PlayingAreaView extends Control {

    public PlayingAreaView() {
        setSkin(createDefaultSkin());
    }

    @Override
    protected Skin<?> createDefaultSkin() {
        return new PlayingAreaSkin(this);
    }
}
